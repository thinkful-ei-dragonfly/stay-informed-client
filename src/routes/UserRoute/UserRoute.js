
import React, { Component } from 'react';
import UserContext from '../../contexts/UserContext'
// import {Link} from 'react-router-dom';
// import Button from '../../components/Button/Button';

class UserRoute extends Component {
  state = {

  }

  static contextType = UserContext

  componentDidMount() {
    console.log(this.context.address)
  }

  render() {
    return (
      <section className='container'>
          <h3>...</h3>
      </section>
      );
  }
}

export default UserRoute
