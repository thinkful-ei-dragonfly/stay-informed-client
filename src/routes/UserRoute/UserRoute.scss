@import '../../index.scss';
.update-wrapper {
  display: flex;
  justify-content: space-between;

  .update-text,
  .UpdateForm {
    width: 47.5%;
  }

  .UpdateForm {
    .form-fields {
      padding: 25px 0;
      display: flex;
      align-items: flex-start;
      justify-content: space-between;

      label {
        color: $blue;
        width: 20%;
        display: inline-block;
      }
      input {
        width: 75%;
        font-size: 1.5em;
        background: inherit;
        border: 0;
        border-bottom: 2px solid $blue;
        border-style: solid;
        color: $blue;
      }

    }
    button {
      margin-top: 25px;
      transition: opacity .25s ease;
      opacity: 0.85;
      border: none;
      background: $red;
      color: $background;

      &:hover {
        cursor: pointer;
        opacity: 1;
      }
    }
  }
}
