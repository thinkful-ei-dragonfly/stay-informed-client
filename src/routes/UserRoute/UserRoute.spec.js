import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserRoute from './UserRoute'

describe(`UserRoute component`, () => {
  it('renders a form.UserRoute by default', () => {
    const wrapper = shallow(<UserRoute />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
