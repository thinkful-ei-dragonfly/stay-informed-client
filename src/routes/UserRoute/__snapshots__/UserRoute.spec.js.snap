// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRoute component renders a form.UserRoute by default 1`] = `
<section
  className="container"
>
  <h3>
    ...
  </h3>
</section>
`;
